reveal/run.js

```diff
@@ -85,25 +99,22 @@
   }
 
   function enableRemoteMode() {
-    if (window.location.search.match(/[?]/))
-      window.location.search = window.location.search + '&remote';
-    else
-      window.location.search = '?remote';
+    window.location.search += window.location.search.match(/[?]/)
+                           ? '&remote'
+                           : "?remote";
```

Comment:
- Such alignement are difficult to maintain (eg if the variable being assigned changes). Lets leave conventional indentation (ie the last two lines should be indented one more step from the first one).
- The last line should use single quotes, like the rest of the code.